src/Network/Network/Models/AzureFirewallPolicy/PSAzureFirewallPolicyDnsSetting.cs

```diff
@@ -22,7 +22,5 @@ public class PSAzureFirewallPolicyDnsSettings
         public IList<string> Servers { get; set; }
 
         public bool? EnableProxy { get; set; }
-
-        public bool? RequireProxyForNetworkRules { get; set; }
```

Comment:
Besides, we usually announce the breaking change at least 2 sprints prior to the major release, in order not to surprise our customers.

Reply:
This particular argument is related to a feature currently in Preview and we're taking the opportunity to streamline the user experience prior to GA, which is coming up in the beginning of November. 

The service will continue to accept the parameter if provided, but its actual functionality will stop working in the next week or two.